helm/dendrite/.helm-docs/monitoring.gotmpl

```diff
@@ -0,0 +1,22 @@
+{{ define "chart.monitoringSection" }}
+## Monitoring
+
+[![Grafana Dashboard](https://grafana.com/api/dashboards/13916/images/9894/image)](https://grafana.com/grafana/dashboards/13916-dendrite/)
+
+* Works well with [Prometheus Operator](https://prometheus-operator.dev/) ([Helmchart](https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack)) and there setup of [Grafana](https://grafana.com/grafana/), by enabling following values:
+```yaml
+prometheus:
+  servicemonitor:
+    enabled: true
+    labels:
+      release: "kube-prometheus-stack"
+  rules:
+    enabled: true # will deploy alert rules
+    labels:
+      release: "kube-prometheus-stack"
+grafana:
+  dashboards:
+    enabled: true # will deploy default dashboards 
+```
+PS: The labels `release=kube-prometheus-stack` is setup with the helmchart of the Prometheus Operator. For Grafana Dashboards it maybe need scan enable to scan in correct namespaces (or ALL), enabled by `sidecar.dashboards.searchNamespace` in [Helmchart of grafana](https://artifacthub.io/packages/helm/grafana/grafana) (which is part of PrometheusOperator, so `grafana.sidecar.dashboards.searchNamespace`) 
```

Comment:
```suggestion
PS: The label `release=kube-prometheus-stack` is setup with the helmchart of the Prometheus Operator. For Grafana Dashboards it may be necessary to enable scanning in the correct namespaces (or ALL), enabled by `sidecar.dashboards.searchNamespace` in [Helmchart of grafana](https://artifacthub.io/packages/helm/grafana/grafana) (which is part of PrometheusOperator, so `grafana.sidecar.dashboards.searchNamespace`) 
```